.gitignore

```diff
@@ -15,6 +15,9 @@ __pycache__
 build/
 dist/
 
+# venv environment 
+venv/
```

Comment:
The environment should only be created locally and schould not be in the github project.

Reply:
nvm